code/components/citizen-scripting-lua/src/LuaScriptNatives.cpp

```diff
@@ -1224,9 +1224,10 @@ struct LuaNativeWrapper
 
 struct LuaNativeContext : public fxNativeContext
 {
-	LUA_INLINE LuaNativeContext(void*, int numArguments)
+

```

Comment:
That struct inherits [fxNativeContext](https://github.com/citizenfx/fivem/blob/master/code/components/citizen-scripting-core/include/fxNativeContext.h), not one of the other four declarations of almost-the-same-thing. 

The amount of bloat/cruft in this area is quite high.

Reply:
> Instead of renaming a fine parameter name, you can initialize them like this (preferred in C++):
> 
> ```c++
> LUA_INLINE LuaNativeContext(void*, int numArguments)
> 	: numArguments(numArguments)
> 	, numResults(0)
> {
> }
> ```

I originally did this, however it required me to redefine those fields again in the struct.

Reply:
🤦‍♂️I see... just do this->numArguments = numArguments for now then, we'll get to it later

Reply:
done